examples/src/main/pegasus/org/coursera/example/Course.courier

```diff
@@ -13,4 +13,6 @@ record Course {
   description: string?
 
   extraData: AnyData
+
+  courseMetadata: union[CourseMetadata]
```

Comment:
just for testing that relations on unions work. i can remove this field if you want